api/auth/cert/cert.go

```diff
@@ -0,0 +1,165 @@
+package cert
```

Comment:
```suggestion
// Copyright (c) HashiCorp, Inc.
// SPDX-License-Identifier: MPL-2.0

package cert
```
We need license header at the top

Reply:
https://github.com/hashicorp/vault/actions/runs/13307362071/job/37168842965?pr=29546